cmp/grid/GridModel.js

```diff
@@ -171,13 +171,10 @@ export class GridModel {
     /**
      * Export grid data using Hoist's server-side export.
      *
-     * @param {Object} options
-     * @param {string} options.type - type of export - one of ['excel', 'excelTable', 'csv'].
-     * @param {(string|function)} [options.filename] - name for exported file or closure to generate.
+     * @param {Object} options - Export options. See ExportManager.exportAsync() for options.
```

Comment:
A bit problematic that ExportManager is marked as a *private* class.  If that's how we want to handle it for now, we should move the documentation of these options here.  

Other option is to go ahead and promote ExportManager to something like GridExportService.  It would be a little unusual having a service that has a dependency on the GridModel, but perhaps appropriate in this case?  This of course relates to #759